src/components/Button/button.test.js

```diff
@@ -0,0 +1,40 @@
+import Button from '.';
+import { fireEvent, render, screen } from '../../utils/tests/renderWithRouter';
+
+const handleClick = (event) => {
+  event.target.textContent = 'Clicked';
+};
+
+describe('Button Tests', () => {
+  it('Should render correctly', () => {
+    render(<Button>Press ME</Button>);
+    const button = screen.getByRole('button', { name: /press me/i });
+    expect(button).toBeInTheDocument();
+    expect(button).toHaveClass('bg-primary-700');
+  });
+  it('Should render incorrectly when search for value "press me"', () => {
```

Comment:
```suggestion
  it('Should render the children prop correctly"', () => {
```
Here you can replace this test with a new one to check if the children prop is rendered correctly, maybe using `toHaveTextContent()` method

Reply:
It was to handle default content which is 'Button' instead 'press me'

Reply:
I think it's a correct test, but for complement that, i'm going to add new test to handle TextContent as you saw before!